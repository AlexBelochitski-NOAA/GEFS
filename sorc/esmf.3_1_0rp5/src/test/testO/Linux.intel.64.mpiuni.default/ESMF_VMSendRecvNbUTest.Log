20190610 112233.612536 INFO     PET0 Running with ESMF Version 3.1.0rp5
20190610 112233.612799 INFO     PET0  Beginning Test, file ESMF_VMSendRecvNbUTest.F90, line          84
20190610 112233.612817 INFO     PET0  NUMBER_OF_PROCESSORS           1
20190610 112233.612832 INFO     PET0  First round: src=           0  dst=           0
20190610 112233.612870 INFO     PET0  PASS  SendRecvNb local I4 data Test, ESMF_VMSendRecvNbUTest.F90, line         168
20190610 112233.612891 INFO     PET0  PASS  SendRecvNb local R4 data Test, ESMF_VMSendRecvNbUTest.F90, line         179
20190610 112233.612901 INFO     PET0  PASS  SendRecvNb local R8 data Test, ESMF_VMSendRecvNbUTest.F90, line         190
20190610 112233.615060 INFO     PET0  PASS  SendRecvNb local LOGICAL data Test, ESMF_VMSendRecvNbUTest.F90, line         201
20190610 112233.615083 INFO     PET0  PASS  Waiting for I4 SendRecvNb, ESMF_VMSendRecvNbUTest.F90, line         212
20190610 112233.615109 INFO     PET0  PASS  Verify local I4 data after receive Test, ESMF_VMSendRecvNbUTest.F90, line         223
20190610 112233.615179 INFO     PET0  PASS  Waiting for R4 SendRecvNb, ESMF_VMSendRecvNbUTest.F90, line         233
20190610 112233.615239 INFO     PET0  PASS  Verify local R4 data after receive Test, ESMF_VMSendRecvNbUTest.F90, line         245
20190610 112233.615248 INFO     PET0  PASS  Waiting for R8 SendRecvNb, ESMF_VMSendRecvNbUTest.F90, line         255
20190610 112233.615305 INFO     PET0  PASS  Verify local R8 data after receive Test, ESMF_VMSendRecvNbUTest.F90, line         267
20190610 112233.615316 INFO     PET0  PASS  Waiting for LOGICAL SendRecvNb, ESMF_VMSendRecvNbUTest.F90, line         277
20190610 112233.615353 INFO     PET0  PASS  Verify local LOGICAL data after receive Test, ESMF_VMSendRecvNbUTest.F90, line         298
20190610 112233.615356 INFO     PET0  Second round: src=           0  dst=           0
20190610 112233.615374 INFO     PET0  PASS  SendRecvNb local I4 data Test, ESMF_VMSendRecvNbUTest.F90, line         341
20190610 112233.615385 INFO     PET0  PASS  SendRecvNb local R4 data Test, ESMF_VMSendRecvNbUTest.F90, line         352
20190610 112233.615395 INFO     PET0  PASS  SendRecvNb local R8 data Test, ESMF_VMSendRecvNbUTest.F90, line         363
20190610 112233.615422 INFO     PET0  PASS  SendRecvNb local LOGICAL data Test, ESMF_VMSendRecvNbUTest.F90, line         374
20190610 112233.615442 INFO     PET0  PASS  Waiting for all queued non-blocking comms in VM, ESMF_VMSendRecvNbUTest.F90, line         385
20190610 112233.615459 INFO     PET0  PASS  Verify local I4 data after receive Test, ESMF_VMSendRecvNbUTest.F90, line         399
20190610 112233.615476 INFO     PET0  PASS  Verify local R4 data after receive Test, ESMF_VMSendRecvNbUTest.F90, line         413
20190610 112233.615496 INFO     PET0  PASS  Verify local R8 data after receive Test, ESMF_VMSendRecvNbUTest.F90, line         427
20190610 112233.615525 INFO     PET0  PASS  Verify local LOGICAL data after receive Test, ESMF_VMSendRecvNbUTest.F90, line         450
20190610 112233.615539 INFO     PET0  Number of failed tests:           0
20190610 112233.615546 INFO     PET0  Ending Test, file ESMF_VMSendRecvNbUTest.F90, line         453
