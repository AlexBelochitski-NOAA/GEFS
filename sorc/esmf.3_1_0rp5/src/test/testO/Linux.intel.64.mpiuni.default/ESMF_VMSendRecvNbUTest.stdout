  Beginning Test, file ESMF_VMSendRecvNbUTest.F90, line          84
  NUMBER_OF_PROCESSORS           1
  PASS  SendRecvNb local I4 data Test, ESMF_VMSendRecvNbUTest.F90, line         
 168
  PASS  SendRecvNb local R4 data Test, ESMF_VMSendRecvNbUTest.F90, line         
 179
  PASS  SendRecvNb local R8 data Test, ESMF_VMSendRecvNbUTest.F90, line         
 190
  PASS  SendRecvNb local LOGICAL data Test, ESMF_VMSendRecvNbUTest.F90, line    
      201
  PASS  Waiting for I4 SendRecvNb, ESMF_VMSendRecvNbUTest.F90, line         212
           0  After rcv i_recvData is            1           2 ( should be 
           1           2  )
  PASS  Verify local I4 data after receive Test, ESMF_VMSendRecvNbUTest.F90, lin
 e         223
  PASS  Waiting for R4 SendRecvNb, ESMF_VMSendRecvNbUTest.F90, line         233
           0 After recv r4_recvData is    1.000000       2.000000     
 ( should be    1.000000       2.000000      )
  PASS  Verify local R4 data after receive Test, ESMF_VMSendRecvNbUTest.F90, lin
 e         245
  PASS  Waiting for R8 SendRecvNb, ESMF_VMSendRecvNbUTest.F90, line         255
           0 After recv r8_recvData is    1.00000000000000     
   2.00000000000000      ( Should be    1.00000000000000     
   2.00000000000000       )
  PASS  Verify local R8 data after receive Test, ESMF_VMSendRecvNbUTest.F90, lin
 e         267
  PASS  Waiting for LOGICAL SendRecvNb, ESMF_VMSendRecvNbUTest.F90, line        
  277
           0 After recv: Recv_Logical(1) is False
           0 After recv: Recv_Logical(2) is True
           0 After recv: Logical_soln(1) is False
           0 After recv: logical_soln(2) is True
  PASS  Verify local LOGICAL data after receive Test, ESMF_VMSendRecvNbUTest.F90
 , line         298
  PASS  SendRecvNb local I4 data Test, ESMF_VMSendRecvNbUTest.F90, line         
 341
  PASS  SendRecvNb local R4 data Test, ESMF_VMSendRecvNbUTest.F90, line         
 352
  PASS  SendRecvNb local R8 data Test, ESMF_VMSendRecvNbUTest.F90, line         
 363
  PASS  SendRecvNb local LOGICAL data Test, ESMF_VMSendRecvNbUTest.F90, line    
      374
  PASS  Waiting for all queued non-blocking comms in VM, ESMF_VMSendRecvNbUTest.
 F90, line         385
           0  After rcv i_recvData is            1           2 ( should be 
           1           2  )
  PASS  Verify local I4 data after receive Test, ESMF_VMSendRecvNbUTest.F90, lin
 e         399
           0 After recv r4_recvData is    1.000000       2.000000     
 ( should be    1.000000       2.000000      )
  PASS  Verify local R4 data after receive Test, ESMF_VMSendRecvNbUTest.F90, lin
 e         413
           0 After recv r8_recvData is    1.00000000000000     
   2.00000000000000      ( Should be    1.00000000000000     
   2.00000000000000       )
  PASS  Verify local R8 data after receive Test, ESMF_VMSendRecvNbUTest.F90, lin
 e         427
           0 After recv: Recv_Logical(1) is False
           0 After recv: Recv_Logical(2) is True
           0 After recv: Logical_soln(1) is False
           0 After recv: logical_soln(2) is True
  PASS  Verify local LOGICAL data after receive Test, ESMF_VMSendRecvNbUTest.F90
 , line         450
  Number of failed tests:           0
  Ending Test, file ESMF_VMSendRecvNbUTest.F90, line         453
